### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by staticcat** [tcpexmachina/remy#20]
Currently there is no license file, or license attached to the source. As a result this makes this project cool, but can't look at it for fear of implementing something which is protected.

**Comment by jbenet** [comment]
Agreed here. I'd love to use Remy in [IPFS](https://github.com/jbenet/ipfs) but cannot unless it is a BSD, MIT license.

**Comment by fxfactorial** [comment]
Is there any more relevancy of this issue?
